### DISCUSSION TITLE
stepper enable?

### DISCUSSION THREAD

**Discussion by bestbean** [kachurovskiy/nanoels#60] [General]
If stepper is powered on, it's not easy to turn lead screw manually.
Even if controller is set to half current on stillstand.
So i have a switched plug for stepper power source as a workaround, if manual adjusting is needed, and also for powersaving.

Would using stepper enable / disable an option?
May be if nanoELS is in "off" state, stepper could also be disabled.

**Comment by kachurovskiy** [comment]
Good call, yes, I should have added support for the ENA pin. I'm afraid it's a little late now - though there should be free pins on Arduino that could be used for that that can be connected with a jumper wire.

**Comment by kachurovskiy** [comment]
Stepper ENA is now appropriately controlled in NanoEls H2 - https://github.com/kachurovskiy/nanoels/tree/main/h2
